generator/generator.yml

```diff
@@ -101,3 +101,41 @@ modules:
       - 1.3.6.1.4.1.1718.3.2.2  # infeedTable
       - 1.3.6.1.4.1.1718.3.2.3  # outletTable
 
+# A10 Ax Load Balancers
+#
+# The MIB files are available for download through the load balancer GUI:
+# - Hover over System on the menu bar, then select Monitoring.
+# - Click on the SNMP tab, then select SNMP MIB Download from the drop-down menu.
+# - Select a target location for the MIB archive file, the click Save.
+#
+# A10 Networks doesn't share them publicly.
+# 
+  a10_axlb:
+    walk:
+      - sysUpTime # Uptime
+      - interfaces # Interfaces 
+      - axSysCpu # CPU stats
+      - axSysMemory # Memory stats
+      - axSysDisk # Disk stats
+      - axSysHwInfo # Hardware stats (Temperature, Fans Speed)
+      - axNetwork # Network stats
+
+# Palo Alto Firewalls
+#
+# Palo Alto MIBs can be found here: 
+# https://www.paloaltonetworks.com/documentation/misc/snmp-mibs.html
+# 
+# PanOS 7.1 enterprise MIBs:
+# https://www.paloaltonetworks.com/content/dam/pan/en_US/assets/zip/technical-documentation/snmp-mib-modules/PAN-MIB-MODULES-7.1.zip
+#
+  paloalto_fw:
+    walk: 
+      - sysUpTime # Uptime
+      - interfaces # Interfaces 
+      - hrDevice 
+      - hrSystem 
+      - hrStorage 
+      - 1.3.6.1.4.1.25461.2.1.2.1 # system objects
```

Comment:
Correct what @SuperQ said. Do I need to change them to use names instead of OIDs?